### PR TITLE
The WAR! Update

### PR DESCRIPTION
**Pull request by DVDPlayerOfDiscordFame**
### **Sides chosen as Demo-Soldier tension mounts!**

In this PR, I commit several changes to the existing weapons pool, as well as add in two new weapons-- a DMR for Nanotrasen, and a Battlerifle for the Syndicate, to flesh out the firearms pool. These will be my final additions to the weapons ecosystem until new components or toys to play with are added, but for what we have this PR should be a much better representation of the ammo pool and style of weapons than we what we currently have.

All changes to current weapons are made with the intent of carving out a more defined identity and niche for them, so that they can feel more unique from each other and remove some of the redundancy in the selection, and not at all to actually affect any sort of balance in the weapons (with the exception of the Cobra being buffed noticeably, but that's it.) Changes and additions will be listed comprehensively in the changelog.

Other things of semi-relevancy: I have added the conversion kits to the emagged lathe recipe lists, as well as some of their magazines.

The finer change details:
- .20 rifle ammunition damage changed from 17 pierce to 19 pierce (+2 damage)
- .30 rifle ammunition damage changed from 19 pierce to 25 pierce (+6 damage)
- .25 caseless ammunition damage changed from 19 pierce to 21 pierce (+2 damage). This does mean the Cobra gets a noticeable buff, but its damage/dps won't be out of line. DPS from 76 to 84, total mag damage from 190 to 210.
- Lecter rate of fire reduced from 5 to 4.5 (DPS from 85/sec to 85.5/sec)
- L6 SAW ammunition type changed from .30 rifle to .20 rifle (gun will feel and operate exactly the same as before)
- Mosin ammunition type changed from .30 rifle to .20 rifle (gun will feel and operate exactly the same as before)
- AKMS rate of fire reduced from 5 to 3.75 (DPS from 95/sec to 93.75/sec), magazine size reduced from 30 to 23, and has been made less accurate (min/max angle of 1/2 to 2/12 with a gain of 3/sec)

**Changelog**
:cl:
- tweak: .20 rifle, .30 rifle, and .25 caseless each receive damage buffs
- tweak: Lecter and AKMS received firerate reductions to account for damage increases. AKMS received further adjustments.
- tweak: L6 SAW and Mosin changed from .30 rifle to .20 rifle to keep their performance the same as before
- add: Central Command has finalized a deal with Blackburn Munitions Inc. and has outfitted each station with one of their new "Daito" precision firearms-- a new slow-firing weapon chambered in .30 rifle ammunition.
- add: The Syndicate has completed development on their new line of Nemesis-11 Battle Rifles chambered in .25 caseless and will offer them for both traitors and nuclear operatives to choose for their arsenal.

### CODE REVIEW

File: Resources/Locale/en-US/_Impstation/store/uplink-catalog.ftl [hunk 1]
```diff
@@ -1,4 +1,11 @@
...
 uplink-zipper-ap-name = Zipper .22
-uplink-zipper-ap-desc = A regretful purchase from a liquidation sale. Comes loaded with armor-piercing rounds. Uses .22 Auto.
\ No newline at end of file
+uplink-zipper-ap-desc = A regretful purchase from a liquidation sale. Comes loaded with armor-piercing rounds. Uses .22 Auto.
+
+uplink-nemesis-br-name = Nemesis-11 Bundle
+uplink-nemesis-br-desc = Precise and Deadly: the stylish Nemesis-11 Battle Rifle, bundled with 3 magazines.
+
+uplink-rifle-magazine-nemesis-name = Rifle Magazine (.25 Caseless)
+uplink-rifle-magazine-nemesis-desc = Rifle magazine with 24 cartridges. Compatible with the Nemesis-11.
```
Comment (Darkmajia) [inline Resources/Locale/en-US/_Impstation/store/uplink-catalog.ftl:11]: nitpick: github hates it when there isn't an empty line at the end of files, any files with the "no entry" sign could do with a newline. i realise many of these already existed but they might as well get fixed up while they're easily viewable

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@
       tags:
         - CartridgeLightRifle
     proto: CartridgeLightRifle
-    capacity: 30
+    capacity: 23
```
Comment (Darkmajia) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml:17]: 23 feels like a very odd number - i know this is to keep the per-magazine damage in-line with previous values, but it feels strange to me

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml [hunk 3]
```diff
@@ -36,14 +36,17 @@
...
   components:
   - type: Tag
     tags:
       - MagazineLightRifleBox
   - type: BallisticAmmoProvider
-    proto: CartridgeLightRifle
+    whitelist:
+      tags:
+        - CartridgeRifle
+    proto: CartridgeRifle
```
Comment (Darkmajia) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml:49]: i'm not entirely sure about the whole .30 -> .20 thing with this pr, but i'm also not a gun nerd so i dunno. this is probably a good thing?

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml [hunk 4]
```diff
@@ -88,7 +88,7 @@
   name: L6C ROW
   id: WeaponLightMachineGunL6C
   parent: BaseItem
-  description: An L6 SAW for use by cyborgs. Creates .30 rifle ammo on the fly from an internal ammo fabricator, which slowly self-charges.
+  description: A L6 SAW for use by cyborgs. Creates .20 rifle ammo on the fly from an internal ammo fabricator, which slowly self-charges.
```
Comment (Darkmajia) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml:91]: i think this should still be "An"

File: Resources/Prototypes/_Impstation/Catalog/Cargo/cargo_armory.yml [hunk 5]
```diff
@@ -16,4 +16,14 @@
...
   category: cargoproduct-category-name-armory
+  group: market
+
+- type: cargoProduct
+  id: ArmoryDMR
+  icon:
+    sprite: _Impstation/Objects/Weapons/Guns/Rifles/daito.rsi
+    state: icon
+  product: CrateArmoryDMR
+  cost: 10000
```
Comment (Darkmajia) [inline Resources/Prototypes/_Impstation/Catalog/Cargo/cargo_armory.yml:27]: 10k might be a bit expensive considering the wt550s are 9k, but i suppose it's better to err on the side of caution without having experienced these guns in a real environment

File: Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Rifles/dmr.yml [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+      gun_chamber: !type:ContainerSlot
+  - type: MagazineVisuals
+    magState: mag
+    steps: 1
+    zeroVisible: true
+  - type: Appearance
+
+- type: entity
+  name: nemesis-11
+  parent: [BaseWeaponRifle, BaseRestrictedContraband]
```
Comment (Darkmajia) [inline Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Rifles/dmr.yml:64]: this needs `BaseSyndicateContraband` too

**Review by Darkmajia — CHANGES_REQUESTED** [review]

**Comment by Darkmajia** [comment]
added a few comments, but overall - i like it a lot! good job

File: Resources/Audio/_Impstation/Weapons/Guns/Gunshots/daito.ogg
[the remark this answers is not in the document]
Comment (DVDPlayerOfDiscordFame) [reply Resources/Audio/_Impstation/Weapons/Guns/Gunshots/daito.ogg:1]: It's a consequence of whatever the sound engine is in Robust struggling to handle multiple sound effects going on. Like, it starts to get confused when even something as slow-firing as a 3 shot/sec rifle starts shooting, and the beginning of the firing sound (where most of the punch is) starts to get cut off periodically. Similar issue is how a target being gibbed/burnt will completely override the impact sound in favour of the gibbing sound.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml (hunk 2 above)
Comment (DVDPlayerOfDiscordFame) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml:17]: I'm open to ideas for this. If I round its number up to 25, the total magazine damage becomes 625 which is, a LOT. Like WAY more than anything Security has. I could bump it down to 20 to be in-line with the Daito, which is okay I guess. 23 just happens to be that weird prime number comfort zone for number of shots vs. total damage.

Comment (Darkmajia) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml:17]: yeah it's a weird situation. i'll just take this for now and we'll see about adjustments later (probably) (we are going to forget)

**Review by Darkmajia — APPROVED** [review]

**Pull request merged**